tensorflow_lite_support/python/task/vision/pybinds/_pywrap_image_embedder.cc

```diff
@@ -88,22 +97,56 @@ PYBIND11_MODULE(_pywrap_image_embedder, m) {
 
              ASSIGN_OR_RETURN(std::unique_ptr<FrameBuffer> frame_buffer,
                               CreateFrameBufferFromImageData(image_data));
-             return self.Embed(*frame_buffer, vision_bounding_box);
+             ASSIGN_OR_RETURN(EmbeddingResult vision_embedding_result,
+                              self.Embed(*frame_buffer, vision_bounding_box));
+
+             // Convert from vision::EmbeddingResult to
+             // processor::EmbeddingResult
+             processor::EmbeddingResult embedding_result;
+               embedding_result.ParseFromString(
+                       vision_embedding_result.SerializeAsString());
+             return embedding_result;
            })
       .def("get_embedding_by_index",
            [](ImageEmbedder& self,
               const processor::EmbeddingResult& embedding_result,
-              const int index) -> Embedding {
+              const int index)
+              -> tflite::support::StatusOr<processor::Embedding> {
              // Convert from processor::EmbeddingResult to
-             // vision::EmbeddingResult as the later is used in the C++ API.
+             // vision::EmbeddingResult as the latter is used in the C++ API.
              EmbeddingResult vision_embedding_result;
              vision_embedding_result.ParseFromString(
                  embedding_result.SerializeAsString());
-             return self.GetEmbeddingByIndex(vision_embedding_result, index);
+
+             vision::Embedding vision_embedding {
+                 self.GetEmbeddingByIndex(vision_embedding_result, index)};
+             // Convert from vision::Embedding to
+             // processor::Embedding
+             processor::Embedding embedding;
+               embedding.ParseFromString(
+                     vision_embedding.SerializeAsString());
+             return embedding;
            })
       .def("get_number_of_output_layers",
            &ImageEmbedder::GetNumberOfOutputLayers)
       .def("get_embedding_dimension", &ImageEmbedder::GetEmbeddingDimension)
+      .def_static(
+          "cosine_similarity",
+          [](const processor::FeatureVector& u,
+             const processor::FeatureVector& v) {
+              // Convert from processor::FeatureVector to vision::FeatureVector
```

Comment:
@khanhlvg The tests should pass now I believe. Could you check it out on your end?